tests/test_postprocess.py

```diff
@@ -0,0 +1,306 @@
+    """
+    filenames = postprocess.get_image_paths(
+        "./tests/data", "tanzania", tanzania_image_size, "grid_066"
+    )
+    assert np.all([f.endswith(".png") for f in filenames])
+
+
+def test_extract_images(tanzania_image_size, tanzania_nb_output_testing_images):
+    """Test the image extraction function, that retrieve the accurate data in a
+    'numpy.array' starting from a list of filenames
+
+    This image data must be shaped as (nb_filenames, image_size, image_size, 3).
+    """
+    filenames = postprocess.get_image_paths(
+        "./tests/data", "tanzania", tanzania_image_size, "tanzania_sample"
+    )
+    images = postprocess.extract_images(filenames)
+    assert len(images.shape) == 4
+    assert images.shape[0] == tanzania_nb_output_testing_images
+    assert images.shape[1] == tanzania_image_size
+    assert images.shape[2] == tanzania_image_size
+    assert images.shape[3] == 3
+
+
+def test_get_labels(tanzania_image_size, tanzania_nb_labels):
+    """Test the label retrieving from dataset glossary
+    """
+    labels = postprocess.get_labels(
+        "./tests/data", "tanzania", tanzania_image_size
+    )
+    assert len(labels) == tanzania_nb_labels
+    with pytest.raises(ValueError):
+        postprocess.get_labels(
+            "./elsewhere-on-the-disk", "wrong-dataset", tanzania_image_size
+        )
+
+
+def test_get_trained_model(tanzania_image_size, tanzania_nb_labels):
+    """Test the semantic segmentation model retrieving
+
+    Postprocessing implies getting a 'unet' model (i.e. segmentation
+    segmentation), hence the model must be shaped as follows:
+    - an input layer of shape (None, image_size, image_size, 3)
+    - an output layer of shape (None, image_size, image_size, nb_labels)
+    - 69 hidden layers, amonst which:
+      + 57 are related to convolution
+      + 8 are related to upsampling
+      + 4 are related to pooling
+    """
+    model = postprocess.get_trained_model(
+        "./tests.data", "tanzania", tanzania_image_size, tanzania_nb_labels
+    )
+    assert model.input_shape[1:] == (
+        tanzania_image_size, tanzania_image_size, 3
+    )
+    assert model.output_shape[1:] == (
+        tanzania_image_size, tanzania_image_size, tanzania_nb_labels
+    )
+    model_input_layer = [ml for ml in model.layers if "input" in ml.name]
+    assert len(model_input_layer) == 1
+    model_output_layer = [ml for ml in model.layers if "output" in ml.name]
+    assert len(model_output_layer) == 1
+    model_conv_layers = [ml for ml in model.layers if "conv" in ml.name]
+    assert len(model_conv_layers) == 57
+    model_up_layers = [ml for ml in model.layers if "up" in ml.name]
+    assert len(model_up_layers) == 8
+    model_pool_layers = [ml for ml in model.layers if "pool" in ml.name]
+    assert len(model_pool_layers) == 4
+
+
+def test_assign_label_colors():
+    """Test the label colourization function, that allows to replace label IDs
```

Comment:
colorization?

Reply:
You're right. I write the code and the docstrings in US english, however "colourization" comes from UK english. I change that!